run-tests.sh

```diff
@@ -68,6 +69,26 @@ for arch in $ARCHS; do
     for test in $TESTS_SSP; do
         $arch-w64-$target_os-clang $test.c -o $TEST_DIR/$test.exe -fstack-protector-strong
     done
+    for test in $TESTS_UWP; do
+        set +e
+        # compilation should fail for UWP and WinRT
+        $arch-w64-$target_os-clang $test.c -o $TEST_DIR/$test.exe -Wimplicit-function-declaration -Werror
+        UWP_ERROR=$?
+        set -e
+        case $target_os in
+        mingw32uwp|mingw32winrt)
+            if [ $UWP_ERROR -eq 0 ]; then
+                echo "UWP compilation should have failed for test $test!"
+                exit 1
+            fi
+            ;;
+        *)
+            if [ $UWP_ERROR -ne 0 ]; then
```

Comment:
Isn't this flipped? If the compilation succeeded, `-eq 0`, we should add it to `TESTS_EXTRA`. If not, we should actually error out here as well.

Reply:
Indeed, fixed.